apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/CommonCodings.java

```diff
@@ -1,6 +1,11 @@
 package gov.cms.bfd.server.war.commons;
 
```

Comment:
@rcrupp this is just a general comment. Could you additional tests just to verify if there's no test coverage for these getters / where the class gets instantiated

Reply:
👋 Hi, @meliGuzman. We can address you questions outside of this PR if you should still have any. Thanks!